pkg/providers/aws/provider_blobstorage.go

```diff
@@ -156,6 +156,49 @@ func (p *BlobStorageProvider) CreateStorage(ctx context.Context, bs *v1alpha1.Bl
 		},
 	}
 
+	// Adding tags to s3
```

Comment:
Would it be worth, abstracting this logic out into its own `reconcileBucketTagging` function. IMO it would read better and follow the same pattern as reconciling the bucket creation

Reply:
had that in one of the closed pr but had issues with testing due to it not needing credentials to connect to aws.

Reply:
As discussed above using the existing session with credential give a 403. Creating a new session with Credential give a 403. Creating a new session with region without credentials works, but this breaks the tests when in a separate function.